catalog/spatial/wcs/spatial-wcs-common/src/main/java/org/codice/ddf/spatial/ogc/wcs/catalog/GetCapabilitiesRequest.java

```diff
@@ -19,10 +19,13 @@
  */
 public class GetCapabilitiesRequest {
 
+  @SuppressWarnings("squid:S1068") /* Unrecoverable state that we do not want caught later */
```

Comment:
Since the member variables will be automatically injected by the JAX-RS annotations left as is with suppressions